utility/importer/chrome_importer.cc

```diff
@@ -108,29 +111,43 @@ bool SetEncryptionKeyForPasswordImporting(
 }
 #endif
 
-class ScopedCopyFile {
- public:
-  explicit ScopedCopyFile(const base::FilePath& original_file_path) {
-    DCHECK(base::PathExists(original_file_path));
-    if (base::CreateTemporaryFile(&copied_file_path_))
-      copy_success_ = base::CopyFile(original_file_path, copied_file_path_);
-  }
-
-  ~ScopedCopyFile() {
-    if (base::PathExists(copied_file_path_))
-      base::DeleteFile(copied_file_path_, false);
-  }
+bool SetEncryptionKey(const base::FilePath& source_path) {
+#if defined(OS_LINUX)
+  // Set up crypt config.
+  std::unique_ptr<os_crypt::Config> config(new os_crypt::Config());
+  config->product_name = l10n_util::GetStringUTF8(IDS_PRODUCT_NAME);
+  config->should_use_preference = false;
+  config->user_data_path = source_path;
+  OSCrypt::SetConfig(std::move(config));
+  return true;
+#endif
 
```

Comment:
Does macOS prompt for keychain access? I know it will when importing passwords and user can click "cancel". I don't have a mac, so I can't test 😄

Reply:
Yes, I got that prompt.

Reply:
OK great! if you cancel, does it cancel the import? or will it keep going? (ex: if you had bookmarks imported, does it finish importing bookmark but just not do payment). We had a condition before where if you canceled PW import in Muon that it caused a problem

Reply:
Ah! I didn't check canceling scenario. Will check that on macOS.

Reply:
If card info has empty card number, it will be skipped. https://github.com/brave/brave-core/pull/5582/files#diff-add7d6a5262f1cc04de00d5276fd45b6R485
Empty card means decryption is failed or chrome's data is invalid.